GitCommands/Git/GitModule.cs

```diff
@@ -2049,7 +2049,8 @@ public string UnstageFiles(IList<GitItemStatus> files)
         private void UpdateIndex(Lazy<SynchronizedProcessReader> processReader, string filename)
         {
             ////process.StandardInput.WriteLine("\"" + ToPosixPath(file.Name) + "\"");
-            byte[] bytearr = EncodingHelper.ConvertTo(SystemEncoding,
+            byte[] bytearr = EncodingHelper.ConvertTo(
```

Comment:
📝 This is one of the reasons why I'm a big fan of this rule. It automatically resolves the contention between users who want to avoid excess indentation and users who want to align arguments.